bootstrap/utils/utils.go

```diff
@@ -161,3 +167,47 @@ func DeepCopy(src any, dest any) error {
 	}
 	return nil
 }
+
+func LoadFile(path string, contents *any) error {
+	var fileBytes []byte
+	var err error
+
+	lowerPath := strings.ToLower(path)
+
+	if strings.Contains(lowerPath, "http") {
+		resp, err := http.Get(lowerPath)
+		if err != nil {
+			return fmt.Errorf("Could not get remote file")
+		}
+		defer resp.Body.Close()
+
+		if resp.StatusCode >= 400 {
```

Comment:
only the 200s are considered succes codes, but should we expect only 200 since the other 20x's don't apply?
https://umbraco.com/knowledge-base/http-status-codes/#2xx
```suggestion
		if resp.StatusCode >= 300 {
```